deploy/artifacts/proxy.yaml

```diff
@@ -0,0 +1,136 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: crane-proxy
+  namespace: openshift-migration
+data:
+  clusters: '[]'
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: proxy
+  namespace: openshift-migration
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: proxy
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: proxy
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
```

Comment:
I wonder if these permissions are elevated enough for us to consider allowing OLM to create them for us?

This would require us to include the `proxy` ServiceAccounts and (Cluster)Roles in the ClusterServiceVersion to be deployed alongside our operator.

Reply:
Working on it

Reply:
done